src/main/java/org/c4sg/service/impl/OrganizationServiceImpl.java

```diff
@@ -88,4 +106,21 @@ public String getLogoUploadPath(Integer organizationId) {
         return LOGO_UPLOAD.getValue() + File.separator + organizationId + IMAGE.getValue();
     }
 
+	@Override
+	public List<OrganizationDTO> findByUser(Integer userId) {
+		User user = userDAO.findById(userId);
+        requireNonNull(user, "Invalid User Id");
+		List<UserOrganization> userOrganizations = userOrganizationDAO.findByUserId(userId);
+		Collections.sort(userOrganizations, (p1, p2) -> {
+			int result = 0;
+			result = p1.getStatus().compareTo(p2.getStatus());
+			return result*-1;
```

Comment:
Lol! I just copied/cloned this from an existing program that is for the UserProject entity. Looking at the latest changes I think status has been removed which should not have been there in the first place. I think the developer was trying to reverse the result of the comparison meaning it is sorting it out reverse of the natural sorting order. And I don't know why???